### PR TITLE
Disables django commenting within javascript

### PR DESCRIPTION
**Pull request by scuml**
Title says it all.

### CODE REVIEW

File: Default (OSX).sublime-keymap [hunk 1]
```diff
@@ -4,15 +4,15 @@
         "command": "toggle_comment_django",
         "args": { "block": false },
         "context": [
-            { "key": "selector", "operator": "equal", "operand": "text.html.django" },
+            { "key": "selector", "operator": "equal", "operand": "text.html.django & -source.js" },
```
Comment (squ1b3r) [inline Default (OSX).sublime-keymap:7]: What is `-source.js`?

**Comment by scuml** [comment]
Excludes django commenting from within <script> tags in django html templates. On Oct 10, 2022, at 1:14 PM, Andrew Kutyrev ***@***.***> wrote:﻿
@squ1b3r commented on this pull request.



In Default (OSX).sublime-keymap:
> @@ -4,15 +4,15 @@
         "command": "toggle_comment_django",
         "args": { "block": false },
         "context": [
-            { "key": "selector", "operator": "equal", "operand": "text.html.django" },
+            { "key": "selector", "operator": "equal", "operand": "text.html.django & -source.js" },

What is -source.js?

**Comment by squ1b3r** [comment]
Oh, so it's a scope? Looks like an extension so I was confused there for a second.

**Comment by scuml** [comment]
Ya. Just a scope.

**Review by squ1b3r — APPROVED** [review]

**Pull request merged**